pkgs/development/libraries/box2d/default.nix

```diff
@@ -22,6 +24,7 @@ stdenv.mkDerivation rec {
   ];
 
   prePatch = ''
+    find
```

Comment:
```suggestion
```

Reply:
thanks for noticing. stupid mistake

Reply:
It's fine, good to see that this was tested.

Reply:
It's fine, good to see that this was tested.